support/jsdoc/jsdoc-custom.js

```diff
@@ -99,7 +99,7 @@ $(function initSearchBar() {
             location.href = host + 'docs.html#.' + suggestion;
         } else {
             var $el = document.getElementById('.' + suggestion);
-            $('#main').animate({ scrollTop: $el.offsetTop }, 500);
+            $('#main').animate({ scrollTop: $el.offsetTop - 60 }, 500);
```

Comment:
I changed the `h1` top padding to `0` in order to accomplish a left blue vertical bar in each header, but. Now, the spacing between contents is by setting a top `margin`, and `margins` are not used to calculate the `(x,y)` position of the element. Because of that, the scroll was not giving much "space" before (it was perfectly aligned with the top nav bar..)